lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/ts_node_lowering.cpp

```diff
@@ -77,36 +72,35 @@ class TSNodeLowering : public TSNodeLoweringInterface {
           torch::lazy::NodeCast<torch_lazy_tensors::ir::ops::AsStrided>(
               node, torch::lazy::OpKind(at::aten::as_strided)));
     }
-    if (node->op() ==
-        *torch_lazy_tensors::ir::ops::ltc_as_strided_view_update) {
-      return LowerAsStridedViewUpdate(
-          torch::lazy::NodeCast<
-              torch_lazy_tensors::ir::ops::AsStridedViewUpdate>(
-              node, *torch_lazy_tensors::ir::ops::ltc_as_strided_view_update));
+    if (node->op() == torch_lazy_tensors::ir::ops::as_strided_reverse) {
+      return LowerAsStridedReverse(
+          torch::lazy::NodeCast<torch_lazy_tensors::ir::ops::AsStridedReverse>(
+              node, torch_lazy_tensors::ir::ops::as_strided_reverse));
     }
     if (node->op() == *torch_lazy_tensors::ir::ops::ltc_cast) {
       return LowerCast(torch::lazy::NodeCast<torch_lazy_tensors::ir::ops::Cast>(
           node, *torch_lazy_tensors::ir::ops::ltc_cast));
     }
-    if (node->op() == *torch_lazy_tensors::ir::ops::ltc_generic_slice) {
+    if (node->op() == torch_lazy_tensors::ir::ops::generic_slice) {
```

Comment:
i'm wondering.. (and it's fine for this to be a separate PR if we do it) - should we move these lowerings into the node classes?  hm, i guess maybe we can't; i have not read the node classes yet, but since they are not under ts_backend/ it probably doesn't make sense to include lowerings in them.

Reply:
Right, if later we decide to promote TsNode to Node, then it makes sense to make these as node's Lower methods. For now, we probably should just keep them here.